packages/composites/bob/README.md

```diff
@@ -0,0 +1,68 @@
+# Bob adapter for Chainlink
+
+The adapter provides an interface for retrieving data from the source blockchain.
+
+## Configuration
+
+The adapter takes the following environment variables:
+
+| Required? |   Name    |       Description       | Options |       Defaults to       |
+| :-------: | :-------: | :---------------------: | :-----: | :---------------------: |
+|           | `RPC_URL` | Blockchain RPC endpoint |         | `http://localhost:8545` |
+
+This composite adapter incorporates the [`json-rpc`](../../sources/json-rpc) adapter by calling `"method": "getblockchainfo"`.
+
+## Running
+
+See the [Composite Adapter README](../README.md) for more information on how to get started.
+
+---
+
+## Endpoints
+- `endpoint`: The parameter to query for. Default: "format"
+- Currently, there is only one "endpoint:" format. This adapter was designed so that adding new internal endpoints would be easy to implement.
+
+### Format Endpoint:
+The format endpoint encodes the chainId, block hash, and block receiptsRoot as bytes and returns that without a 0x prefix.
+---
+
+### Input Params
+
+| Required? |    Name    |      Description       | Options | Defaults to  |
+| :-------: | :--------: | :--------------------: | :-----: | :----------: |
+|    Yes    | `chainID`  | Parameter to query for |         |              |
```

Comment:
It would be nice to give a little bit more in the description for these two.

Could be as simple as "An identifier for which network of the blockchain to use"